### DISCUSSION TITLE
Internal server error help!

### DISCUSSION THREAD

**Discussion by fei-max** [community/community#20311] [Projects and Issues] [answered]
remote: Internal Server Error
To github.com:cryptogold/______.git
! [remote rejected] main -> main (Internal Server Error)
error: failed to push some refs to '[git@github.com](mailto:git@github.com):cryptogold/uphold-tech-test.git'

**Comment by edhends** [comment, marked as answer]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
